em_workflows/lrg_2d_rgb/flow.py

```diff
@@ -57,7 +57,7 @@ def bioformats_gen_zarr(file_path: FilePath):
         LRG2DConfig.bioformats2raw,
         "--overwrite",
         "--scale-format-string",
-        "%2$d",
+        "'%2$d'",
```

Comment:
I do not believe that the command goes through a shell, so the quotes are not needed here.